webapp-django/crashstats/crashstats/views.py

```diff
@@ -69,6 +69,10 @@
 
 
 def make_correlations_count_cache_key(product, version, platform, signature):
+    assert product
+    assert version
+    assert platform
+    assert signature
```

Comment:
I used it to debug. I took care of the real problem elsewhere later but forgot to delete these lines.